.github/workflows/deploy-to-s3.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy To S3
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.CI_CD_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.CI_CD_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install wrc
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          # install wkhtmltopdf
```

Comment:
Is this a new use of `wkhtmltopdf`? My understanding is that `wkhtmltopdf` is deprecated and should no longer be used.

Reply:
this is not a new usecase, the [wrc](https://github.com/thewca/wca-regulations-compiler) tool requires it for target=pdf